backend/package-lock.json

```diff
@@ -4,1123 +4,4108 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@sindresorhus/is": {
-      "version": "0.14.0",
-      "resolved": "https://registry.npmjs.org/@sindresorhus/is/-/is-0.14.0.tgz",
-      "integrity": "sha512-9NET910DNaIPngYnLLPeg+Ogzqsi9uM4mSboU5y6p8S5DzMTVEsJZrawi+BoDNUVBa2DhJqQYUFvMDfgU062LQ==",
-      "dev": true
-    },
-    "@szmarczak/http-timer": {
-      "version": "1.1.2",
-      "resolved": "https://registry.npmjs.org/@szmarczak/http-timer/-/http-timer-1.1.2.tgz",
-      "integrity": "sha512-XIB2XbzHTN6ieIjfIMV9hlVcfPU26s2vafYWQcZHWXHOxiaRZYEDKEwdl129Zyg50+foYV2jCgtrqSA6qNuNSA==",
+    "@babel/code-frame": {
```

Comment:
what is going on here?

Reply:
@rebelsky do you know what babel is and why it would suddenly be everywhere? @lisa-shens thinks she was required to add all this stuff to do something, but they can't remember what...

Reply:
@smorrison44 Sorry if you've already figured this out/this comment is unnecessary, but I think babel is the library that parses your code (jsx, I believe, since it's not exactly Javascript) and converts it to the type of Javascript that  browsers can interpret. I don't know what babel files are strictly necessary, but I'm guessing it was a case of updating the dependencies? Not sure.